cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -385,9 +385,11 @@ public static String getZKQuorum(CConfiguration cConf) {
    * Spark on k8s
    */
   public static final class Spark {
```

Comment:
Why this is defined in common module? Spark is an extension. Also, it seems like these constants are used by `ArtifactFetcherService`, which is actually only used by the spark module. Can it all be moved inside spark module?